src/controls/qml/ListItem.qml

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2023 - Timo Könnecke <github.com/eLtMosen>
+ *               2015 - Florent Revest <revestflo@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.9
+import QtGraphicalEffects 1.15
+import org.asteroid.controls 1.0
+import org.asteroid.utils 1.0
+
+Item {
+    id: gridItem
```

Comment:
We can keep an id here since it's used (or make onClicked an alias) but it should be corrected to something like listItem rather than gridItem which isn't true anymore

Reply:
I'd prefer making onClicked and alias and eliminating the id here.